ViewModel/ImpactArea/ImpactAreaElement.cs

```diff
@@ -69,30 +69,34 @@ public ImpactAreaElement(string userdefinedname,string description, ObservableCo
 
         public void RemoveElement(object sender, EventArgs e)
         {
-            List<IASElementSet> iasElems = StudyCache.GetChildElementsOfType<IASElementSet>();
-            if (iasElems.Count > 0)
+            MessageBoxResult messageBoxResult = MessageBox.Show("Are you sure you want to delete '" + Name + "'?", "Delete " + Name + "?", MessageBoxButton.YesNo, MessageBoxImage.Question);
```

Comment:
As we discussed, let's make this one dialog per scenario instead of two dialogs if you have multiple IASElementSets